app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<!-- first loop around roasts__view and second loop around roasts__brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<main class="roasts">
+  <section class="roasts__slider-wrapper">
+      <div class="slider-axis" aria-hidden="true">
+        <%= image_tag("light-slider-bean.svg", :alt=>" ") %>
+        <span class="axial-text left-axis">Light</span>
+        <span class="desktop-only"><span></span></span>
+        <span><span></span></span>
+        <span><span></span></span>
+        <span><span></span></span>
+        <span><span></span></span>
+        <span><span></span></span>
+        <span><span></span></span>
+        <span class="desktop-only"><span></span></span>
+        <span class="axial-text right-axis">Dark</span>
+        <%= image_tag("dark-slider-bean.svg", :alt=>" ") %>
+      </div>
+      <div class="slider">
+        <%= image_tag("light-slider-bean.svg", :alt=>" ") %>
+        <label class="screen-reader-only" for="roast-range">Roast Level: Light to Dark</label>
+        <input type="range" id="roast-range" value="0" min="0" max="7" step="1" class="slider__range" data-feature="sliderToggle">
+        <%= image_tag("dark-slider-bean.svg", :alt=>" ") %>
+      </div>
+  </section>
+    <% roasts.each_with_index do |roast, index| %>
+      <% roast_index = index %>
+      <section class = "roasts__view" id = "roast<%= index %>">
```

Comment:
remove extra spaces around ` = `.  That's mal-formed HTML and may cause some cross-browser issues.